psi4/driver/procrouting/proc.py

```diff
@@ -1377,11 +1377,21 @@ def scf_helper(name, post_scf=True, **kwargs):
         for pv in ["SCF TOTAL ENERGY", "CURRENT ENERGY", "CURRENT REFERENCE ENERGY"]:
             obj.set_variable(pv, e_scf)
 
-    # We always would like to print a little dipole information
-    if kwargs.get('scf_do_dipole', True):
+    # We always would like to print a little property information
+    if kwargs.get('scf_do_properties', True):
         oeprop = core.OEProp(scf_wfn)
         oeprop.set_title("SCF")
-        oeprop.add("DIPOLE")
+
+        # Figure our properties, if empty do dipole
+        props = [x.upper() for x in core.get_option("SCF", "SCF_PROPERTIES")]
```

Comment:
My knowledge of this machinery is a little outdated.  Is there a reason to do this via options, rather than just pass the `props` array directly in place of the `scf_do_properties` bool?